libethcore/ChainOperationParams.cpp

```diff
@@ -66,3 +66,16 @@ u256 ChainOperationParams::u256Param(string const& _name) const
 
 	return u256(fromBigEndian<u256>(fromHex(at)));
 }
+
+u256 ChainOperationParams::blockReward(bool isByzantiumOrLater) const
+{
+	if (isByzantiumOrLater)
+		return u256(3 * (bigint(10) ^ 18)); // 3 ETH
```

Comment:
I think there should also be a global constant called `ether` somewhere with the right value.